style.css

```diff
@@ -127,3 +127,668 @@ li {
   text-align: left;
   padding: 20px;
 }
+
+.grid-container {
+  display: grid;
+  padding: 10px;
+  position: relative;
+  width: 375px;
+  height: 3000px;
+  background: #fff;
+  top: 900px;
+}
+
+.head {
+  position: static;
+  width: 327px;
+  height: 83px;
+  left: 1px;
+  top: 1px;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #1c1a19;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: -20px;
+}
+
+.border {
+  position: inherit;
+  width: 327px;
+  height: 1px;
+  left: 1px;
+  top: 10px;
+  border-bottom: 1px solid #212121;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: -20px;
+}
+
```

Comment:
- [ ] Good job styling the whole container with grid. But according to the requirements, you should also style each card with `flexbox`. I can see you used flex to style the card tags, but you should consider it to style everything.